skopt/space/space.py

```diff
@@ -214,6 +214,9 @@ def inverse_transform(self, Xt):
     def bounds(self):
         return (self.low, self.high)
 
+    def __contains__(self, point):
+        return self.low <= point <= self.high
```

Comment:
are these the right comparison operators? (We should document this in the doc string)